src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ CHANGELOG
  * Make the `config/` directory optional in `MicroKernelTrait`, add support for service arguments in the
    invokable Kernel class, and register `FrameworkBundle` by default when the `bundles.php` file is missing
  * [BC BREAK] The `secrets:decrypt-to-local` command terminates with a non-zero exit code when a secret could not be read
+ * Wire the `Symfony\Component\PropertyInfo\Extractor\ConstructorExtractor` class
```

Comment:
this line should be removed IMO (the next line describes the relevant user-visible change)

Reply:
Done